pipelines/build/common/openjdk_build_pipeline.groovy

```diff
@@ -405,23 +405,51 @@ class Build {
-                                            context.string(name: 'LABEL_ADDITION', value: additionalTestLabel),
-                                            context.string(name: 'KEEP_REPORTDIR', value: "${keep_test_reportdir}"),
-                                            context.string(name: 'PARALLEL', value: parallel),
-                                            context.string(name: 'NUM_MACHINES', value: "${numMachinesPerTest}"),
-                                            context.booleanParam(name: 'USE_TESTENV_PROPERTIES', value: useTestEnvProperties),
-                                            context.booleanParam(name: 'GENERATE_JOBS', value: aqaAutoGen),
-                                            context.string(name: 'ADOPTOPENJDK_BRANCH', value: aqaBranch),
-                                            context.string(name: 'ACTIVE_NODE_TIMEOUT', value: "${buildConfig.ACTIVE_NODE_TIMEOUT}"),
-                                            context.booleanParam(name: 'DYNAMIC_COMPILE', value: DYNAMIC_COMPILE)]
+                            def testJob = context.build job: jobName,
+                                            propagate: false,
+                                            parameters: [
+                                                context.string(name: 'UPSTREAM_JOB_NUMBER', value: "${env.BUILD_NUMBER}"),
+                                                context.string(name: 'UPSTREAM_JOB_NAME', value: "${env.JOB_NAME}"),
+                                                context.string(name: 'JDK_REPO', value: jdkRepo),
+                                                context.string(name: 'JDK_BRANCH', value: jdkBranch),
+                                                context.string(name: 'OPENJ9_BRANCH', value: openj9Branch),
+                                                context.string(name: 'LABEL_ADDITION', value: additionalTestLabel),
+                                                context.string(name: 'KEEP_REPORTDIR', value: "${keep_test_reportdir}"),
+                                                context.string(name: 'PARALLEL', value: parallel),
+                                                context.string(name: 'NUM_MACHINES', value: "${numMachinesPerTest}"),
+                                                context.booleanParam(name: 'USE_TESTENV_PROPERTIES', value: useTestEnvProperties),
+                                                context.booleanParam(name: 'GENERATE_JOBS', value: aqaAutoGen),
+                                                context.string(name: 'ADOPTOPENJDK_BRANCH', value: aqaBranch),
+                                                context.string(name: 'ACTIVE_NODE_TIMEOUT', value: "${buildConfig.ACTIVE_NODE_TIMEOUT}"),
+                                                context.booleanParam(name: 'DYNAMIC_COMPILE', value: DYNAMIC_COMPILE)],
+                                            wait: true
+                            //for testing, merge should be if (buildConfig.RELEASE)
```

Comment:
do we want this as ! RELEASE still ?

Reply:
Good catch! Updated. This will limit to release only.